device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/sai.profile

```diff
@@ -3,3 +3,4 @@ hwId=et6448m
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/profile.ini
 switchProfile=/usr/share/sonic/hwsku/SAI-M0-48x1G-4x10G.xml
 createSwitchTimeout=90
+customEVlanUpdate=1
```

Comment:
Can you help me understand why need the EVlan Update setting?

Reply:
Fix is triggered in SAI implementation based on the flag setting.